### ISSUE TITLE
Suggestion: Provide a mechanism to inject a link to a CSS file

### ISSUE THREAD

**Issue by mbtyomkin** [Microsoft/PowerBI-JavaScript#100]
Due to limitations in how PowerBI reports can be styled from within PowerBI Desktop app, and due to how CSS interacts with iframe contents, our ability to style the dashboards to match desired look and feel are limited. 
Proposal: provide callers with an ability to inject some css or inject a link to a css file so that it can be loaded within the iframe.

**Comment by joemensor** [comment]
I think this would be a great implementation, we should have some control over the look and feel of the dashboard.

**Comment by t-val** [comment]
Big thumbs up to this idea.  I would love to change the light gray report container background to white as well as change the filters slider color to match our branding.

**Comment by jonbiere** [comment]
I agree. I am experiencing some sizing issues while embedding a smaller report onto my web app and I've isolated the bug down to the iframe's css. Sadly i cannot overwrite this because it is an iframe originating from a different domain. If i had the ability to inject my own css I could solve my issue...

**Comment by robrien6** [comment]
Agreed, this would represent a huge improvement in the look and feel of embedded reports, especially when viewed on different devices.

**Comment by robrien6** [comment]
Hi, 

Is there any timeframe on when this might be delivered? Is it being worked on?

**Comment by dm044** [comment]
Any updates on this. Have the same issue of a gray background and would like to inject some css to change the background.
